capstone/src/main/java/com/step902020/capstone/Individual.java

```diff
@@ -41,17 +44,10 @@ public Individual(Long timestamp, String firstName, String lastName, String emai
     this.university = university;
     this.userType = userType;
     this.image = image;
-  }
-
-  public Individual(long datastoreId, Long timestamp, String firstName, String lastName, String email, String university, String userType, String image) {
-    this.datastoreId = datastoreId;
-    this.timestamp = timestamp;
-    this.firstName = firstName;
-    this.lastName = lastName;
-    this.email = email;
-    this.university = university;
-    this.userType = userType;
-    this.image = image;
+    savedEvents = new HashSet<String>();
+    savedEvents.add("hello 1");
+    savedEvents.add("hello 2");
+    savedEvents.add("hello 3");
```

Comment:
Do you still need the hard coded saved events?

Reply:
Yes for now. So far it doesn't seem like the "Event" entity is complete yet.

Reply:
Oh sorry I meant that since you're storing a set<String>, you can populate the list with the String values from the save-event input box so you wouldn't need to hardcode strings. but since it'll temp anyways it doesn't matter which way you choose!